eng/templates/setup-template.yml

```diff
@@ -0,0 +1,19 @@
+# setup-template.yml
```

Comment:
Could we add a comment to say that this affects both windows and linux builds here please? It should be apparent from code but wouldn't hurt to have multiple indicators.